srcpkgs/gyazo/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'gyazo'
 pkgname=gyazo
-version=1.2.1
+version=1.3.1
 revision=1
+archs="*"
 wrksrc="Gyazo-for-Linux-${version}"
 depends="ruby xclip ImageMagick"
-archs=noarch
 short_desc="Seriously Instant Screen-Grabbing"
-maintainer="DirectorX <DirectorX@users.noreply.github.com>"
-license="GPL-2"
+maintainer="DirectorX <void.directorx@protonmail.com>"
```

Comment:
Are you @DirectorX ? Otherwise I'd leave it to them to fix the email.

Reply:
Yes I am

Reply:
I have changed the maintainer to myself, however, this would be the first package for me to maintain. I am able to respond to any emails and Github notifications.